src/open_inwoner/js/components/anchor-menu/scroll_to_top.js

```diff
@@ -0,0 +1,19 @@
+// Get the button
+const mybutton = document.getElementById('scroll-anchor')
```

Comment:
Just a general remark, we use camel-case names in js `(myButton).` No need of changes here.

Reply:
@vaszig Thanks, it seemed weird to use camelCase here because CSS is case-insensitive, but after some investigation, it seems it's 'convention' to use camelCase with ID's.